frontend/src/components/groups-create/components/users-table/users-table.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  FC,
+  FormControl,
+  FormControlErrors,
+  UsersGetResponseDto,
+} from 'common/types/types';
+import { Table } from 'components/common/common';
+import { useMemo } from 'hooks/hooks';
+import { Column } from 'react-table';
+
+import { GroupCreationFieldsName } from '../../common/enums/enums';
+import { GroupCreationUsersTableRow } from '../../common/types/types';
+import { getUserColumns } from '../../helpers/helpers';
+import styles from './styles.module.scss';
+
+type Props = {
+  users: UsersGetResponseDto[];
+  onCheckboxToggle: (value: number) => void;
+  useFormData: {
+    control: FormControl;
+    errors: FormControlErrors;
+  };
+};
+
+const UsersTable: FC<Props> = ({ users, onCheckboxToggle, useFormData }) => {
+  const columns = useMemo<Column<GroupCreationUsersTableRow>[]>(() => {
+    return getUserColumns(useFormData.control, {
+      name: GroupCreationFieldsName.USER_IDS,
+      errors: useFormData.errors,
+      onCheckboxToggle,
+    });
+  }, []);
```

Comment:
With `useMemo` but without dependencies on values provided from the outside - cols won't respond on changes made by with this values 
```suggestion
  const columns = useMemo<Column<GroupCreationUsersTableRow>[]>(() => {
    return getUserColumns(useFormData.control, {
      name: GroupCreationFieldsName.USER_IDS,
      errors: useFormData.errors,
      onCheckboxToggle,
    });
  }, [useFormData]);
```

Reply:
something with gh, I can't see those changes 🤔

Reply:
Oh, haven't pushed that)
Wait a minute)